src/components/schedule-visualizer/lift.tsx

```diff
@@ -0,0 +1,211 @@
+import { makeStyles } from '@material-ui/core';
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+import React from 'react';
+import { RomiCoreLift } from './lift-overlay';
+import Door from './door/door';
+import { UpArrow, DownArrow } from './arrow';
+
+export enum LiftMotionStates {
+  STOPPED,
+  UP,
+  DOWN,
+  UNKNOWN,
+}
+
+export enum LiftModeStates {
```

Comment:
Can we reuse the constants from `romi-js-core-interfaces`?

Reply:
Done! Thanks!